packages/gauge/test/gauge_general_test.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:test/test.dart';
+
+void main() {
+  final String gaugeRootPath = Directory.current.absolute.path;
+
+  test('help works', () {
+    final ProcessResult result = Process.runSync(
+      'dart',
+      <String>['$gaugeRootPath/bin/gauge.dart', 'help'],
+    );
+
+    expect(
+        result.stdout.toString(),
```

Comment:
nit: add a trailing comma after container(...) and format it like the other expects below.